engine/runtime/src/main/java/org/enso/interpreter/node/expression/builtin/meta/EqualsNode.java

```diff
@@ -82,8 +81,7 @@ public static EqualsNode getUncached() {
    * @param other the other object
    * @return {@code true} if {@code self} and {@code that} seem equal
    */
-  public boolean execute(
-      VirtualFrame frame, @AcceptsError Object self, @AcceptsError Object other) {
+  public boolean execute(VirtualFrame frame, Object self, Object other) {
```

Comment:
If a builtin method does not handle `DataflowError` itself, it should not annotate parameters with `@AcceptsError`. In other words, if `@AcceptsError` annotation is used, there should be a specialization that handles DataflowErrors.